src/runner.ts

```diff
@@ -193,8 +193,8 @@ export class Runner {
         try {
             this.processFiles(onComplete, files, program);
         } catch (error) {
-            if (error.name === FatalError.NAME) {
-                console.error(error.message);
+            if ((error as FatalError).name === FatalError.NAME) {
```

Comment:
No, TS doesn't allow a type annotation there. It wouldn't be safe anyway since TS doesn't have checked exceptions.